src/crypto/hash.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { getTransactionBytes } from './../transactions/transactionBytes';
+
+/**
+ * @method getSha256Hash
+ * @param stringToSign
+ * @param format
+ *
+ * @return {string}
+ */
+export function getSha256Hash(stringToSign, format) {
+	if (typeof stringToSign === 'object') {
```

Comment:
If someone passes a normal object (not a `Buffer`) or e.g. an `Array`, do we want to pass that directly to the hash function?